docker-compose-dev-django.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  django:
+    restart: always
+    build:
+      context: ./backend
+      dockerfile: Dockerfile-dev
+    environment:
+      - DEBUG=1
+      - DATABASE_URL=postgres://postgres@db:5432/postgres
+    command: tail -f /dev/null
```

Comment:
instead of having a separate file, why not just override the entry point of the other container?

```
docker-compose run --help

--entrypoint CMD      Override the entrypoint of the image.

```